src/components/team/Teammate.jsx

```diff
@@ -1,5 +1,31 @@
+import React from "react";
+import Image from "next/image";
+import Graves from "public/team/graves.png";
+
 const Teammate = () => {
-  return <div>Teammate</div>;
+  return (
+    <div className="bg-gradient-to-r from-hlg-blue-300 to-hlg-black">
+      <div className="flex ">
+        <Image
+          src={Graves} // Path to your image inside the 'public' directory
+          alt="Graves Image"
+        />
+
+        <div className="mt-10 pl-10">
+          <div className="bg-gradient-to-r from-hlg-blue-100 to-hlg-blue-200 bg-clip-text text-transparent">
+            <div className="font-bold leading-tight text-5xl m-0 p-0">
```

Comment:
the font will be too big when we put it all together, it'll look like this, can you change it to 3xl instead of 5xl
<img width="1054" alt="image" src="https://github.com/acm-ucr/hlg-website/assets/90938120/2b3ca570-1108-4baa-a1bf-f47a1aaaef71">